### DISCUSSION TITLE
Directus does not discover existing tables

### DISCUSSION THREAD

**Discussion by Zyles** [directus/directus#19769] [Feature Requests]
### Summary

I am testing Directus with supabase cloud. After installation in supabase I created a table.

Then I installed Directus via docker and connected it to supabase cloud.

The existing tables however are not visible. It did create the directus_ tables so everything is connected.

I can create a new model and it shows up.

Shouldn't it be able to discover the supabase created tables?

### Basic Example

Create supabase cloud project. And connect to it.

### Motivation

Use supabase + Directus

### Detailed Design

Not much details.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

x

### Alternatives

x

### Adoption Strategy

x

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request. We use [Discord](https://directus.chat) for community Q&A 🙂

**Reply by Zyles** [reply]
Mostly being ignored there.

Having a forum would be better to actually get a response once someone has the time and effort.

**Reply by rijkvanzanten** [reply]
> Having a forum would be better to actually get a response once someone has the time and effort.

I don't see how a forum style would automatically mean the response time would improve. The core team wouldn't magically have more resources available to answer every question if it was in a forum style somewhere else, and from experience in our own community Discord has worked better than GH Discussions in having members in the community help each other 🤷🏻
